### PR TITLE
Add system status badge documentation to web console config

### PR DESCRIPTION
**Pull request by rhamilto**
Words are hard.  ;-)  @jwforres, I kept this in line with what we do for online, but maybe it should be more generic?

### CODE REVIEW

**Comment by jwforres** [comment]
I think it would be good to explain what is generally supported, and then we can say this is an example of using statuspage.io to generate the system status

so explain the extension-registry extension point that is supported, and that if they want to use our existing styles they should use such and such markup and give an html snippet

**Comment by rhamilto** [comment]
Take 2.  Attempted to improve the description and opted to make the example very simple so it's just an icon and configurable url.  Better, @jwforres?

**Comment by jwforres** [comment]
I'm fine with that, it does leave out the "5 issues" text but im not sure how helpful that is anyway.

File: install_config/web_console_customization.adoc [hunk 1]
```diff
@@ -505,6 +505,67 @@ ifdef::openshift-enterprise[]
...
+  .run([
+    'extensionRegistry',
+    function(extensionRegistry) {
+      // Replace http://status.example.com/ with your domain
+      var system_status_elem = $('<a href="http://status.example.com/"' +
+      'target="_blank" class="nav-item-iconic system-status"><span title="' +
+      'System Status" class="fa status-icon pficon-warning-triangle-o">' +
+      '</span></a>');
+
+      extensionRegistry
```
Comment (jwforres) [inline install_config/web_console_customization.adoc]: may want a comment here that says you should only add the extension point if you want the status icon to show up, a typical extension will optionally add this based on the existence of issues after hitting a status API.

**Comment by rhamilto** [comment]
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1519738

Attention @ahardin-rh

**Comment by jwforres** [comment]
@ahardin-rh we will want to get it into the 3.7 docs as well i think

**Comment by rhamilto** [comment]
@ahardin-rh, can this be merged so I can move https://bugzilla.redhat.com/show_bug.cgi?id=1519738 to MODIFIED?

**Comment by ahardin-rh** [comment]
@rhamilto Yes, so sorry for the delay. Not sure how I missed this. 😱

**Comment by rhamilto** [comment]
@ahardin-rh, no worries.  Thanks!

**Comment by ahardin-rh** [comment]
Revision history stated in follow-up PR: https://github.com/openshift/openshift-docs/pull/7093

**Pull request merged**